.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,52 @@
-#### done when
+## Summary
+Brief description of the changes in this PR.
 
-- [ ] Implement
-- [ ] Fix: #
+## Related Issue
+- Fixes #[issue number]
+- Related to #[issue number]
+
+## Type of Change
```

Comment:
Type of change is superfluous, remove.
Can this template overall be more of a placeholder?